test/regressions/site/index.html

```diff
@@ -6,8 +6,6 @@
       name="viewport"
       content="width=device-width, initial-scale=1, user-scalable=0, maximum-scale=1, minimum-scale=1"
     >
-    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700,400italic" rel="stylesheet" type="text/css">
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
> We could always delay until the font is loaded

Yes, that's exactly what I'm doing. Oh, I haven't though about having them localy. We would have to keep them up to date.